config/services_test.yaml

```diff
@@ -0,0 +1,9 @@
+services:
+    _defaults:
+        public: true
```

Comment:
nit pick - but this isn't technically needed in projects - you can fetch private services out of a test container with `self::$container->get(...)`